charts/system/kubelet-csr-approver/values.yaml

```diff
@@ -1,18 +1,117 @@
 image:
-  repository: tccr.io/tccr/alpine
+  repository: ghcr.io/postfinance/kubelet-csr-approver
   pullPolicy: IfNotPresent
-  tag: latest@sha256:ade0065e19edaa4f6903d464ee70605111a48394536deb94f31b661264704558
+  tag: 1.2.2@sha256:fdccaa3f2e4f59001b99357565bc5995393c53b21074da769fa53620b5138b85
 
 service:
   main:
-    enabled: false
     ports:
       main:
-        enabled: false
+        port: 8080
+        targetPort: 8080
+        protocol: http
 
 workload:
   main:
-    enabled: false
+    enabled: true
+    spec:
+      containers:
+        main:
+          args:
+            - -metrics-bind-address
+            - ":{{ .Values.metrics.port }}"
+            - -health-probe-bind-address
+            - ":8081"
+            - -leader-election
+            - -level
+            - -0
+          probes:
+            liveness:
+              path: "/healthz"
+            readiness:
+              path: "/healthz"
+            startup:
+              path: "/healthz"
+
+# Required configuration item
+providerRegex: ""
+# optional, specified as a string (enclosed with ""). if left empty, defaults to 367 days
+maxExpirationSeconds: ""
+# optional, permits to bypass dns resolution checks. default: false
+bypassDnsResolution: false
+# number of DNS SAN names allowed in a certificate request. defaults to 1
+allowedDnsNames: 1
+# optional, permits ignoring CSRs with another Username than `system:node:...`
+ignoreNonSystemNode: false
+# set this parameter to true to ignore mismatching DNS name and hostname
+bypassHostnameCheck: false
+# optional, list of IP (IPv4, IPv6) subnets that are allowed to submit CSRs
+providerIpPrefixes: []
+#   - 192.168.8.0/22
+#   - fc00::/7
+
+# logging level ranges from -5 (Fatal) to 10 (Verbose). default level is 0
+loggingLevel: 0
```

Comment:
hence these have to be merged with the configmap to the env section, not defined here.
Try to keep comments though!